src/utils/AudioCompare.js

```diff
@@ -24,6 +24,8 @@ const AudioRecorder = (props) => {
   }, []);
 
   const startRecording = async () => {
+    const micStartTime = new Date().getTime();
+    localStorage.setItem("micStart", micStartTime);
```

Comment:
Is this required to store in local-storage?

Reply:
Yes, if we don't use localstorage it will impact on performance of the app.